config/tomcat.yml

```diff
@@ -40,5 +40,5 @@ redis_store:
   timeout: 2000
   connection_pool_size: 2
 geode_store:
-  version: 1.11.+
+  version: 1.11.0
```

Comment:
Do you mind adding a comment above this line that explains that patch versions of the client can be incompatible with certain server versions? It's a bit unexpected and I imagine in the future we will forget why we did this and be tempted to change it back.

Reply:
Sure! After discussing this question with veteran members of the Geode community, I was informed that it is possible for a newer patch version of the Geode client to not be compatible with an older patch version of the Geode server (if a protocol is changed, for example).  We'd like to download the earliest supported version of geode store to make sure we are compatible with all supported TGF4VMs versions.